BUILD.gn

```diff
@@ -18,6 +18,18 @@ executable("aquarium") {
     "src/common/AQUARIUM_ASSERT.h",
     "src/common/FPSTimer.cpp",
     "src/common/FPSTimer.h",
+    "src/common/imgui/imconfig.h",
+    "src/common/imgui/imgui.h",
+    "src/common/imgui/imgui.cpp",
+    "src/common/imgui/imgui_draw.cpp",
+    "src/common/imgui/imgui_demo.cpp",
+    "src/common/imgui/imgui_internal.h",
+    "src/common/imgui/imgui_widgets.cpp",
+    "src/common/imgui/imstb_rectpack.h",
+    "src/common/imgui/imstb_textedit.h",
+    "src/common/imgui/imstb_truetype.h",
+    "src/common/imgui/examples/imgui_impl_glfw.h",
+    "src/common/imgui/examples/imgui_impl_glfw.cpp",

```

Comment:
To mimic the directory hierarchy of Chromium, I think you should update `DEPS` and `.gitignore`, and check in these files in `third_party/imgui` instead.